.github/workflows/run_tests.yml

```diff
@@ -42,12 +44,10 @@ jobs:
         flake8 ./panflute --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
     - name: Download Pandoc
       run: |
-        url="https://github.com/jgm/pandoc/releases/latest"
-        path=$(curl -L $url | grep -o '/jgm/pandoc/releases/download/.*-amd64\.deb')
-        downloadUrl="https://github.com$path"
-        file=${path##*/}
-        wget $downloadUrl &&
-        sudo dpkg -i $file
+        pip install requests
+        downloadUrl=$(python -c 'import requests, sys; v = sys.argv[1]; url = "https://api.github.com/repos/jgm/pandoc/releases/" + ("" if v == "latest" else "tags/") + v; print(next(i["browser_download_url"] for i in requests.get(url).json()["assets"] if i["name"][-3:] == "deb"))' ${{ matrix.pandoc-version }})
```

Comment:
almost feel that this is so complex is should be its own script that is called? Rather than a hard-to-read line.

Reply:
The before and after scripts are both written by me. The former regex the html directly, the later uses the GitHub API itself. While it might have readability issue, it is rarely the thing you need to debug as it is stable. (I've been using both in various projects for years.)

I wouldn't worry about this too much. The main point is to reintroduce multi Python and pandoc versions for testing. I think when I first set up Travis CI for panflute it has much more combinations.